docs/source/_sidebar.yaml

```diff
@@ -4,22 +4,39 @@ switcher:
     - label: v0
       latest: true
       href: ./
+defaultOpenDepth: 2
 items:
-  - label: "Welcome"
+  - label: "Overview"
     href: "."
   - label: "Quickstart"
     href: "./quickstart"
   - label: "Install"
     href: "./install"
-  - label: "User Guide"
-    href: "./guides"
+  - label: "Run the MCP Server"
+    href: "./run"
+  - label: "Define tools"
+    href: "./define-tools"
+  - label: "Configuration"
+    children:
+      - label: "YAML Config Reference"
+        href: "./config-file"
+      - label: "Custom Scalars"
+        href: "./custom-scalars"
+  - label: "Debugging"
+    href: "./debugging"
+  - label: "Deployment"
+    href: "./deploy"
   - label: "Authorization"
     href: "./guides/auth"
```

Comment:
Should this file be moved to a base path since it doesn't show up in the Guides parent?

Reply:
Yes good call!